### DISCUSSION TITLE
Erfaringer med Windows og Jøkul (Gatsby)

### DISCUSSION THREAD

**Discussion by BogdanItumX** [fremtind/jokul#3940] [Spørsmål] [answered]
Hei,

Vi installerte Jøkul på flere Windows-maskiner (Windows 10 og Windows 11) og opplevde at det kan ta 1 time (!) å bygge portalen (med pnpm boot). Det virker som det er kompileringen av Gatsby-filer som tar mest tid (rekorden var 5000 sekunder, og vi har ikke fått det til å gå raskere enn 2500 sekunder).

PC-ene har ganske solid maskinvare (mye minne, gode SSD-disker), og filsystemet er NTFS, som vi mistenker kan være en ulempe sammenlignet med Linux/MacOS, med tanke på at det bare tar et par sekunder der.

Det er ingen feilmeldinger, og verbose-modus tilbyr ingen tips. Skal teste litt med å oppgradere forskjellige avhengigheter og Gatsby, men dette er erfaringer med eksisterende avhengighetsversjoner...

Har dere noen erfaringer med å kjøre Jøkul på ikke-Mac?

![compile gatsby files tok 2715.220 sekunder](https://github.com/user-attachments/assets/3fb398b4-4e67-49cd-af3f-917cbfdd22cd)

**Comment by ivarni** [comment]
Hei,

Vi er i en prosess med å erstatte Gatsby med en Remix/Sanity combo så det har ikke vært prioritert å holde ting oppdatert det siste året. Så vidt jeg vet brukes Windows-maskiner i veldig liten grad blant frontendere i organisasjonen så der er det nok ikke mye erfaring å hente dessverre.

**Reply by BogdanItumX** [reply]
Hei, takk skal du ha.

Noe E.T.A. når Remix/Sanity kombo kommer ute?

**Reply by ivarni** [reply, marked as answer]
Per akkurat nå er vi litt usikre på hvor det kommer i backloggen. Vi har en portal kjørende men det er en del arbeid med å få den gamle ut fra jokul-repoet og den nye inn.
